Sources/App/CRP/JiraService+CRP.swift

```diff
@@ -187,12 +207,15 @@ extension JiraService {
             self.environments = environments
             self.releaseType = releaseType
             self.targetDate = targetDate
-            self.businessImpact = FieldType.TextArea.Document(text: "TBD")
+            self.changeScope = FieldType.TextArea.Document(text: "TBD")
             self.jiraReleaseURL = "\(jiraBaseURL)/secure/Dashboard.jspa?selectPageId=15452"
             self.githubReleaseURL = "https://github.com/\(release.repository.fullName)/releases/tag/\(release.appName)/\(release.version)"
             self.testing = FieldType.TextArea.Document(text: "TBD")
```

Comment:
Another pre-populated request:

```suggestion
            self.testing = FieldType.TextArea.Document(text: "Test Plan - 

[Android & iOS native mobile apps PED](https://docs.google.com/document/d/1GlvBD7DL0B24WOdky_sCJp3bewmwuHHkYfYcrPWDQEI/edit#heading=h.1jdzrbj14q2r) 

TestRail milestone (automated & manual test runs) -   

CircleCI branch pipeline (automated unit tests and build) - 

Internal release notes/QA sign-off -")
```

Reply:
resolved in 7249e1e